create_test_instance.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2016 Google LLC All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from google.cloud.spanner_v1 import Client
+
+
+client = Client(
+    project=os.getenv("GOOGLE_CLOUD_PROJECT", "emulator-test-project")
+)
+
+instance = client.instance("google-cloud-django-backend-tests")
+created_op = instance.create()
+created_op.result(30)  # block until completion
```

Comment:
This script will be runned before tests

Reply:
Instance creation is a long operation, so we're setting 30 secs timeout. Instance clearing is made inside our Django code.